docs/csharp/tutorials/console-webapiclient.md

```diff
@@ -1,388 +1,355 @@
 ---
-title: Create a REST client using .NET Core
-description: This tutorial teaches you about some features in .NET Core and the C# language.
-ms.date: 01/09/2020
-ms.assetid: 51033ce2-7a53-4cdd-966d-9da15c8204d2
+title: "Tutorial: Make HTTP requests in a .NET console app using C#"
+description: Learn how to make HTTP requests to a REST web service and deserialize JSON responses. This tutorial creates a .NET console and uses C#.
+ms.date: 04/21/2021
 ---
 
-# REST client
+# Tutorial: Make HTTP requests in a .NET console app using C\#
 
-This tutorial teaches you a number of features in .NET Core and the C# language. You'll learn:
+This tutorial builds an app that issues HTTP requests to a REST service on GitHub. The app reads information in JSON format and converts the JSON into C# objects. Converting from JSON to C# objects is known as *deserialization*.
 
-* The basics of the .NET Core CLI.
-* An overview of C# Language features.
-* Managing dependencies with NuGet
-* HTTP Communications
-* Processing JSON information
-* Managing configuration with Attributes.
+The tutorial shows how to:
 
-You'll build an application that issues HTTP Requests to a REST
-service on GitHub. You'll read information in JSON format, and convert
-that JSON packet into C# objects. Finally, you'll see how to work with
-C# objects.
+> [!div class="checklist"]
+>
+> * Send HTTP requests.
+> * Deserialize JSON responses.
+> * Configure deserialization with attributes.
 
-There are many features in this tutorial. Let's build them one by one.
-
-If you prefer to follow along with the [final sample](https://github.com/dotnet/samples/tree/main/csharp/getting-started/console-webapiclient) for this article, you can download it. For download instructions, see [Samples and Tutorials](../../samples-and-tutorials/index.md#view-and-download-samples).
+If you prefer to follow along with the [final sample](https://github.com/dotnet/samples/tree/main/csharp/getting-started/console-webapiclient) for this tutorial, you can download it. For download instructions, see [Samples and Tutorials](../../samples-and-tutorials/index.md#view-and-download-samples).
```

Comment:
nit: Samples browser is more suitable when it comes to downloading single samples.

Reply:
I'd agree. We could address this in a separate PR, as that would update the samples repo.